### DISCUSSION TITLE
Appended a simple ICMP rule to all.rules but it is not visible(Securityonion-Suricata)

### DISCUSSION THREAD

**Discussion by kunalhiremath** [Security-Onion-Solutions/securityonion#9176] [Unsupported Versions] [answered]
I am working with Securityonion suricata and I appended an simple ICMP rule to suricata's all.rules file but after sometime it got disappeared. Does anyone know how to make the rule to persist.

**Comment by dougburks** [comment, marked as answer]
Did you follow the instructions in the documentation?
https://docs.securityonion.net/en/2.3/local-rules.html#nids

**Comment by kunalhiremath** [comment]
Thank you very much Doug.
